config/metrics.yml

```diff
@@ -15,67 +15,44 @@ metadata:
 - iterationsPerNamespace
 - jobIterations
 - jobType
-- maxWaitTimeout
 - metricsClosing
-- name
-- preLoadPeriod
-- qps
 - verifyObjects
 - waitForDeletion
 - waitWhenFinished
 - k8sVersion
 - masterNodesCount
 - masterNodesType
-- metricName
 - ocpMajorVersion
 - ocpVersion
 - passed
 - platform
-- publish
 - region
 - sdnType
 - totalNodes
-- uuid
 - workerArch
 - workerNodesCount
 - workerNodesType
+job_summary_filters:
+  platform.keyword: AWS
+  jobConfig.name.keyword: cluster-density-v2
 metrics:
-- alert
-# - avg-ro-apicalls-latency
-# - avg-mutating-apicalls-latency
-# - 99thEtcdDiskBackendCommit
-# - 99thEtcdDiskWalFsync
-# - 99thEtcdRoundTripTime
-# - cpu-kube-controller-manager
-# - max-memory-sum-kube-controller-manager
-# - cpu-etcd
-# - max-memory-sum-etcd
-- cpu-kube-apiserver
-- max-memory-sum-kube-apiserver
-- cpu-masters
-- max-memory-sum-masters
-- cpu-workers
-- max-memory-sum-workers
-- podLatencyQuantilesMeasurement
-benchmark: cluster-density-v2
-exclude_normalization:
-- LatencyMeasurement
-- jobSummary
-- nodeRoles
-- nodeStatus
-- alert
-- -start
-target_filters_by_data:
-- platform: AWS
-target_field_extract_filters:
-# - "^avg-ro-apicalls-latency": '^avg-ro-apicalls-latency.*_byLabelScope_resource(?:_|$).*'
-# - "^avg-ro-apicalls-latency": '^avg-ro-apicalls-latency.*_byLabelScope_namespace(?:_|$).*'
-# - "^avg-mutating-apicalls-latency": '^avg-mutating-apicalls-latency.*_byLabelScope_resource(?:_|$).*'
-# - "^avg-mutating-apicalls-latency": '^avg-mutating-apicalls-latency.*_byLabelScope_namespace(?:_|$).*'
-- "^podLatencyQuantilesMeasurement": 'podLatencyQuantilesMeasurement_Ready_P99'
-# target_fields_to_reduce:
```

Comment:
Also there are often times where we want to reduce same prefixed fields with multiple labels into a singular value (.i.e similar to what we do in our grafana dashboard). For that we need this.


For example 
```
Avg(avg-ro-apicalls-latency_byLabelScrop_resource_pods, avg-ro-apicalls-latency_byLabelScrop_resource_services) --> avg-ro-apicalls-latency_byLabelScrop_resource
```